engine-precompiles/src/xcc.rs

```diff
@@ -52,6 +53,7 @@ mod consts {
     pub(super) const ERR_DELEGATE: &str = "ERR_INVALID_IN_DELEGATE";
     pub(super) const ROUTER_EXEC_NAME: &str = "execute";
     pub(super) const ROUTER_SCHEDULE_NAME: &str = "schedule";
+    pub(super) const TRANSFER_FROM_SELECTOR: [u8; 4] = [0x23, 0xb8, 0x72, 0xdd];
```

Comment:
This needs an explanation of what this is.

Reply:
Done in https://github.com/aurora-is-near/aurora-engine/pull/572/commits/07e0afc5548a637c48d763fb05dcc49c84b033a8